defs/bitcoin/stakenet.json

```diff
@@ -0,0 +1,43 @@
+{
+    "coin_name": "Stakenet",
+    "coin_shortcut": "XSN",
+    "coin_label": "Stakenet",
+    "website": "https://stakenet.io",
+    "github": "https://github.com/X9Developers/XSN",
+    "maintainer": "<support@stakenet.io>",
+    "curve_name": "secp256k1",
+    "address_type": 76,
+    "address_type_p2sh": 16,
+    "maxfee_kb": 2000000,
+    "minfee_kb": 1000,
+    "signed_message_header": "DarkCoin Signed Message:\n",
+    "hash_genesis_block": "00000c822abdbb23e28f79a49d29b41429737c6c7e15df40d1b1f1b35907ae34",
+    "xprv_magic": 76066276,
+    "xpub_magic": 76067358,
+    "xpub_magic_segwit_p2sh": 77429938,
+    "xpub_magic_segwit_native": 78792518,
+    "bech32_prefix": "xc",
+    "cashaddr_prefix": null,
+    "slip44": 199,
+    "segwit": true,
+    "decred": false,
+    "fork_id": null,
+    "force_bip143": false,
+    "bip115": false,
+    "default_fee_b": {
+      "Low": 10,
+      "Economy": 70,
+      "Normal": 140,
+      "High": 200
+    },
+    "dust_limit": 1000,
+    "blocktime_seconds": 60,
+    "uri_prefix": "stakenet",
+    "min_address_length": 27,
+    "max_address_length": 47,
+    "bitcore": [],
+    "blockbook": [
+      "https://xsnexplorer.io"
```

Comment:
remove this, xsnexplorer is not a valid blockbook backend

Reply:
It isn't, it just supports the `/api/<txhash>` that is required by trezorctl for signing transactions.

If you still think we should remove it, we'll do it.

Reply:
I really doubt it implements the same API as our blockbook if it is not the blockbook.

Reply:
It doesn't, it just has the necessary endpoint for signing transactions with trezorctl.

Reply:
@matejcik what do you think?

Reply:
For example, call `GET https://xsnexplorer.io/api/tx/36000e057671b7fc37621bb1a8a547a487198b3cb55e4d8f402abc70e6ecae79` and you'll see that it returns the necessary fields for signing a transaction with trezorctl, `version`, `vin` `vout`, `locktime`.

Reply:
Using those values I got this transaction generated from trezorctl: https://xsnexplorer.io/transactions/1236786df0c03f03ef74061130017646c8d04bb746a978c3e8555e1c0a0026a7

Screenshot:
![img](https://media.discordapp.net/attachments/530427000981028884/543586646726017034/trezor.png)

Reply:
In fact, thinking a bit more, the unknown is whether this would affect other projects, in the case of trezorctl, the behavior is different when the blockbook is there.

Unless you are confident that this won't affect other projects, I think it is better to remove the blockbook.

Reply:
the "blockbook" field is also used in Trezor Wallet, if we enabled this, and it would fail.

given that some changes are expected here - removal of the blockbook functionality from trezorctl, change of the field format for backends - i wouldn't add it right now.

Reply:
@JonSalazar remove the blockbook entry and I'll merge

Reply:
@matejcik @prusnak I ended up wrapping a small PR for supporting custom explorers like ours: https://github.com/trezor/python-trezor/pull/358

Reply:
@prusnak the blockbook links are removed now.